src/components/vue-form-renderer.vue

```diff
@@ -73,6 +73,8 @@ Vue.component('custom-css', {
 
 Vue.use(VueDeepSet);
 
+let Validator = require('validatorjs');
```

Comment:
Nitpick: Since `Validator` isn't re-assigned, you can use `const` instead of `let`.